Add support for end to end tests with Playwright

cmfcmf opened the pull request:
@tom95 you might need to change how you start Roc & Roll on the server, since `server.roc-and-roll.js` moved from `dist` to `dist/production`.

tom95 commented:
Looking great! Surprising amount of complexity for the server setup but looks to make sense.

One pitfall: if you just run `yarn e2e-test` you get a cryptic missing file in a dist folder error, until you find out that first a `yarn e2e-build` is needed. I'll leave an approve here but it should probably still be added to the README, if this is indeed the intended way of approaching it.

cmfcmf commented:
> One pitfall: if you just run `yarn e2e-test` you get a cryptic missing file in a dist folder error, until you find out that first a `yarn e2e-build` is needed. I'll leave an approve here but it should probably still be added to the README, if this is indeed the intended way of approaching it.

Whoops - thank you for noticing that. I added a better error message and adjusted the README.
